<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_13">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="ball_tab">
        <attribute name="title">
         <string>Ball</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>13</x>
           <y>11</y>
           <width>292</width>
           <height>62</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="ball_size_label">
              <property name="text">
               <string>Size:     </string>
              </property>
              <property name="buddy">
               <cstring>ball_size_slider</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="ball_size_slider">
              <property name="minimum">
               <number>5</number>
              </property>
              <property name="maximum">
               <number>30</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ball_size_edit">
              <property name="inputMask">
               <string/>
              </property>
              <property name="text">
               <string>1.0</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="ball_speed_label">
              <property name="text">
               <string>Speed: </string>
              </property>
              <property name="buddy">
               <cstring>ball_speed_slider</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="ball_speed_slider">
              <property name="minimum">
               <number>5</number>
              </property>
              <property name="maximum">
               <number>30</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ball_speed_edit">
              <property name="inputMask">
               <string/>
              </property>
              <property name="text">
               <string>1.0</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="player_tab">
        <attribute name="title">
         <string>Player</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>13</x>
           <y>11</y>
           <width>292</width>
           <height>62</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="player_size_label">
              <property name="text">
               <string>Size:     </string>
              </property>
              <property name="buddy">
               <cstring>player_size_slider</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="player_size_slider">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="minimum">
               <number>5</number>
              </property>
              <property name="maximum">
               <number>30</number>
              </property>
              <property name="sliderPosition">
               <number>10</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="player_size_edit">
              <property name="inputMask">
               <string/>
              </property>
              <property name="text">
               <string>1.0</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="player_speed_label">
              <property name="text">
               <string>Speed: </string>
              </property>
              <property name="buddy">
               <cstring>player_speed_slider</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="player_speed_slider">
              <property name="minimum">
               <number>5</number>
              </property>
              <property name="maximum">
               <number>30</number>
              </property>
              <property name="sliderPosition">
               <number>10</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="player_speed_edit">
              <property name="inputMask">
               <string/>
              </property>
              <property name="text">
               <string>1.0</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="sound_tab">
        <attribute name="title">
         <string>Sound</string>
        </attribute>
        <widget class="QCheckBox" name="mute_check_box">
         <property name="geometry">
          <rect>
           <x>3</x>
           <y>46</y>
           <width>61</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Mute</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>3</x>
           <y>13</y>
           <width>287</width>
           <height>27</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="sound_label">
            <property name="text">
             <string>Sound:</string>
            </property>
            <property name="buddy">
             <cstring>sound_slider</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sound_slider">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="sliderPosition">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="sound_edit">
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string>100</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="controls_tab">
        <attribute name="title">
         <string>Controls</string>
        </attribute>
        <widget class="QRadioButton" name="keyboard_mode_radio_button">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>33</y>
           <width>133</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Keyboard mode</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>62</y>
           <width>269</width>
           <height>28</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="score_to_win_label">
            <property name="text">
             <string>Score to win:</string>
            </property>
            <property name="buddy">
             <cstring>score_to_win_spin_box</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="score_to_win_spin_box">
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="value">
             <number>11</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="endless_check_box">
            <property name="text">
             <string>Endless game</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>2</y>
           <width>245</width>
           <height>25</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="mouse_mode_radio_button">
            <property name="text">
             <string>Mouse mode</string>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>(Accelerating ball)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>48</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Options</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Options</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
